applications/serial_lte_modem/src/slm_at_host.c

```diff
@@ -848,35 +847,50 @@ bool verify_datamode_control(uint16_t time_limit, uint16_t *min_time_limit)
 	return true;
 }
 
-struct at_param_list *slm_get_at_param_list(const char *at_cmd)
+int slm_get_at_param_list(const char *at_cmd, struct at_param_list *list)
 {
 	int err;
 
-	at_params_list_clear(&slm_at_param_list);
-	err = at_parser_params_from_str(at_cmd, NULL, &slm_at_param_list);
+	list = &at_host_param_list;
+
+	at_params_list_clear(list);
```

Comment:
Unnecessary. From `at_cmd_parser.h`:
```
@p list must be initialized. It can be reused to parse multiple commands.
* When calling this function, the list is cleared.
```